src/@next/Bar/BarStyle.ts

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+import { Device, ScreenSize } from '../..';
+import { Neutral } from '../utilities/colors';
+import { space8 } from '../utilities/spacing';
+
+export const StyledBar = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding: 20px 0;
+  width: 100%;
+  position: absolute;
+
+  &[data-align='bottom'] {
+    bottom: 0;
+  }
+  &[data-align='top'] {
+    top: 0;
+  }
+
+  box-shadow: 0px 0px 0px 1px rgba(63, 63, 68, 0.05),
+    0px 1px 3px rgba(63, 63, 68, 0.15);
+  background: ${Neutral.B100};
+`;
+
+export const StyledBarHeaderWrapper = styled.div`
+  display: flex;
+  flex-grow: 1;
+  flex-direction: column;
+`;
+
+export const StyledBarActionWrapper = styled.div`
+  display: flex;
+  flex-direction: column-reverse;
+  align-items: flex-end;
+`;
+
+export const StyledBarContainer = styled.div`
+  position: relative;
+  margin: 0 auto;
+
+  @media ${Device.mobileS} {
```

Comment:
we currently only have 2 Breakpoints defined for new design system. you can checkout how `${Breakpoints.large}` is used in the other new components